packages/variable-editor/src/components/variables/master/VariablesMaster.tsx

```diff
@@ -16,7 +18,6 @@ import { getCoreRowModel, useReactTable, type ColumnDef } from '@tanstack/react-
 import type { DataContext, ValidationMessages } from '../../../protocol/types';
 import { deleteFirstSelectedRow, useTreeGlobalFilter } from '../../../utils/tree/tree';
 import { treeNodeNameAttribute, type TreePath } from '../../../utils/tree/types';
-import { Control } from '../../control/Control';
```

Comment:
did you forgot to remove the control component? :)

Reply:
oops :laughing: